docs/04-Patterns/02-learning-goals.adoc

```diff
@@ -19,7 +19,19 @@ Softwarearchitekt:innen kennen Methoden, um technische von fachlichen Aufgaben d
 * Container Management durch Operator bzw. Controller
 
 [[LZ-4-2]]
-==== LZ 4-2: Passende Resilienzmuster zur Erhöhung von Fehlertoleranz auswählen
+==== LZ 4-2: Passende Technologien für den Betrieb von Modulen auswählen
+
+Softwarearchitekt:innen können geeignete Technologien zum Betrieb von Modulen eines verteilten Systems auswählen, z.B. mit Functions as a Service (FaaS) und Container Orchestrierung.
+
+Darüber hinaus können Sie die Anwendung dieser Technologien Anforderungsgetrieben anwenden. Dabei gibt es verschiedene Aspekte zu beachten wie:
+
+* Skalierungsanforderungen und Ausführungszeit
```

Comment:
Include Ausführungszeit also the time span to start up the function?

Reply:
I included the time to startup in commit: [f9adf03](https://github.com/isaqb-org/curriculum-cloudinfra/pull/25/commits/f9adf03586e9f58e7a56ebad9b8363986ed59ee0)